examples/optics/concave_spherical_mirror.py

```diff
@@ -0,0 +1,69 @@
+from sympy import Symbol, solve, Eq, simplify
+
+from symplyphysics import print_expression, Quantity, prefixes, units, \
+    convert_to
+from symplyphysics.laws.optics import lens_focus_from_object_and_image as focus_law
+from symplyphysics.laws.optics import focal_length_of_a_concave_spherical_mirror as mirror_law
+
+# Example from https://www.test-uz.ru/solutions_view.php?cat=6&num=15.5
+
+# In a concave mirror with a radius of curvature R = 40 cm, they want to get a real image,
+# the height of which is half the height of the object itself.
+# Where should the object be placed and where will the image be obtained?
+
+heigth_object = Symbol("heigth_object")
+heigth_image = Symbol("heigth_image")
+
+# Because it said in example question
+heigth_object_eq = Eq(heigth_object, 2 * heigth_image)
+
+focus = Symbol("focus")
+radius_curvature = Symbol("radius_curvature")
+
+distance_from_object = Symbol("distance_from_object")
+distance_from_image = Symbol("distance_from_image")
+
+heigth_object_value = heigth_object_eq.rhs
+
+# From picture in example:
+# heigth_image / heigth_object  = focus / (distance_from_object - focus)
+similarity_sides_triangle_eq = Eq(heigth_image / heigth_object, focus / (distance_from_object - focus))
```

Comment:
I do not see this equation being used in the solution. Do we really need it?

Reply:
You can use https://github.com/blackyblack/symplyphysics/blob/master/symplyphysics/laws/optics/linear_magnification_from_distance_to_object_and_distance_to_image.py and https://github.com/blackyblack/symplyphysics/blob/master/symplyphysics/laws/optics/linear_magnification_from_object_height_and_image_height.py to get the same equation `h1/h2 = a1/a2`